src/modules/game.js

```diff
@@ -0,0 +1,37 @@
+import { setKey, getKey } from './keys.js';
+import { setScore, getScore } from './scores.js';
+import render from './display.js';
+
+const createNewGame = async () => {
+  const game = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+    },
+    body: JSON.stringify({ name: 'MicroGameXtrem' }),
+  });
+  return game.json().then((res) => {
+    const key = toString(/\w{20}/.exec(res.result)[0]);
+    setKey(key);
+  });
+};
```

Comment:
:red_circle: : [REQUIRED] If you want to create a game you should follow [these instructions](https://microverse.notion.site/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3). You can an API platform like Postman to interact with the Leaderboard API service. The service will respond to your request with a specific unique GameID. You should store the generated game-id in a variable and you can later use it in every request where it is needed.

I think the game should be created only once. There is no need to create a game every time you want to play add or fetch score.

Reply:
> Correction

I create the method to extract the game key and store it in a constant that I later import for use in the urls.